### DISCUSSION TITLE
Is there an effort to get this into the Fedora Project yum repo?

### DISCUSSION THREAD

**Discussion by Jared-Sprague** [trapexit/mergerfs#1379] [Q&A]
I love this project and I use Fedora. It would be great to be able to `dnf install mergerfs` on Fedora.  Has this been looked into or an effor in progress?

**Comment by trapexit** [comment]
I've not been a Redhat ecosystem user for almost 30 years. If someone could point me to how packages get officially included I can look into it but quick searches over the years have not brought up anything obvious on how to do that. I'm not sure I want to be responsible to be a maintainer for a package I would never use not being a user of the platform but if it isn't much work then I suppose I can.

**Reply by Jared-Sprague** [reply]
I'll do some research and ask around the community and update with what I find.

**Reply by trapexit** [reply]
I did just find this. https://docs.fedoraproject.org/en-US/package-maintainers/New_Package_Process_for_New_Contributors/

Sounds a bit involved but if I get some time I can look into it.
